operations/power_management/Prepare_the_System_for_Power_Off.md

```diff
@@ -307,8 +307,8 @@ HPE Cray EX System Admin Toolkit (SAT) product stream documentation (`S-8031`) f
     If active sessions are running, either wait for them to complete or cancel the session. See the following step.
 
     **`NOTE`** If the Compute Rolling Upgrade Service (CRUS) is not being used to manage compute rolling upgrades, then this error can be ignored.
-    If `cray-crus` pods have failed to initialize, the `GET request` will fail. A system using PBS Pro or Slurm where the workload manager scheduler
-    is not running in a pod on a worker node can lead to this failure to initialize. CRUS will be removed in a future release.
+    If `cray-crus` pods have failed to initialize, the `GET request` will fail. A system using PBS Pro or Slurm in which the workload manager scheduler
+    is not running in a pod on a worker node, will fail to initialize. CRUS will be removed in a future release.
```

Comment:
My initial comment here was inaccurate. The subject "system" and the verb phrase "will fail to initialize" need the WLM details to be accurate, so no commas are required. It did lead you to replace "where" with an interrogative pronoun (in which) and it makes the sentence better because it is more precise than "where." With that said, there is no need for the comma after "worker node."

Reply:
Fixed